packages/battery/battery/example/android/app/src/main/AndroidManifest.xml

```diff
@@ -2,11 +2,9 @@
     package="io.flutter.plugins.batteryexample">
    <application
         android:label="batteryexample"
-        android:name="${applicationName}"
         android:icon="@mipmap/ic_launcher">
         <activity
             android:name="io.flutter.embedding.android.FlutterActivity"
-            android:exported="true"
```

Comment:
Nit: You should include the exported="true" as this will need to be update din future versions anyways

Reply:
Presumably it'll have an auto-migration at the point where it becomes required though. This is removed relative to the previous upload because 2.5 doesn't have it in the register, and this is now a 2.5-created example.